packages/react-components/react-datepicker-compat/stories/DatePicker/DatePickerDateRange.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Field, makeStyles } from '@fluentui/react-components';
+import { DatePicker, DateRangeType } from '@fluentui/react-datepicker-compat';
+
+const useStyles = makeStyles({
+  control: {
+    maxWidth: '300px',
+  },
+});
+
+export const DateRange = () => {
+  const styles = useStyles();
+
+  return (
+    <Field label="Select a date">
+      <DatePicker
+        calendar={{ dateRangeType: DateRangeType.Week }}
+        placeholder="Select a date..."
+        className={styles.control}
+      />
+    </Field>
+  );
+};
+
+DateRange.parameters = {
+  docs: {
+    description: {
+      story:
+        'DatePicker allows you to set the selection range type. The range can be `Day`, `Month`, `Week`, and' +
```

Comment:
nit: maybe add examples in this story for all the options so I can easily see them all? Similar to how we show all the appearances for Button next to each other.